events/reporter.go

```diff
@@ -17,6 +17,9 @@ var reporter *EventReporter
 // we use a mutex to ensure thread safety
 var mu sync.RWMutex
 
+// EventQueueSize is a parameterized length of the channel buffers
+const EventQueueSize int = 30 //TODO: replace this with reading from a file
```

Comment:
```suggestion
const EventQueueSize int = 30 // TODO: move this to config
```

Please open an issue to track this and link it here